packages/uploadthing/src/convex.ts

```diff
@@ -58,13 +58,10 @@ export const addUploadthingRoutes = <TRouter extends FileRouter>(
       if (!isCorsRequest) {
         return new Response();
       }
-      if (!process.env.CLIENT_ORIGIN) {
-        throw new Error("Convex deployment doesn't have CLIENT_ORIGIN set");
-      }
       return new Response(null, {
         status: 200,
         headers: {
-          "Access-Control-Allow-Origin": process.env.CLIENT_ORIGIN,
+          "Access-Control-Allow-Origin": "*",
```

Comment:
I think this works as long as there's no authentication header or cookie, do we have an auth header in one of these? if so we have to echo back the origin received in the request instead of using "*" here.

Reply:
My mistake, when setting auth headers manually this is fine!